src/screens/DetailedViewScreen.tsx

```diff
@@ -125,17 +122,91 @@ const DetailedViewScreen: React.FC<PlaceProps> = ({ setPage, placeID }: PlacePro
 						>
 							{distanceInMi ? `${distanceInMi} mi away` : ""}
 						</BodyText>
-						{Object.entries(namesToFieldsMap).map(([attrName, rating], index) => {
-							const attribute = attrName;
-							const score = avgRatingLookup(rating);
```

Comment:
I see that we're not using `avgRatingLookup` anymore, can we delete it?

Reply:
good call, done @carpenterd777